src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -23,10 +29,24 @@ public class TestJobs2dPatterns {
 	 * @param application Application context.
 	 */
 	private static void setupPresetTests(Application application) {
-		SelectTestFigureOptionListener selectTestFigureOptionListener = new SelectTestFigureOptionListener(
-				DriverFeature.getDriverManager());
+		ActionListener selectTestFigureOptionListener1 = new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent actionEvent) {
+				DrawerFeature.getDrawerController().clearPanel();
+				FiguresJoe.figureScript1(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
+
+		ActionListener selectTestFigureOptionListener2 = new ActionListener() {
+			@Override
+			public void actionPerformed(ActionEvent actionEvent) {
+				DrawerFeature.getDrawerController().clearPanel();
+				FiguresJoe.figureScript2(DriverFeature.getDriverManager().getCurrentDriver());
+			}
+		};
```

Comment:
It is good to keep a convention used in the original code (class instance instead of anonymous class).
I see unnecessary code duplication.